tasks/install.yml

```diff
@@ -1,6 +1,14 @@
 ---
+- name: Update APT Cache
+  apt:
+    cache_valid_time: 600
```

Comment:
I don't think it's going to fix the CI, as molecule does an apt update when creating the dockers, but then does some cleanup. And correct me if I'm wrong, but forcing a cache update here can be a bad practice if you don't update the system (maybe you'll pull a package of unbound relying on a an updated library and you have the old one installed).

Maybe we can just force the cache update (with a `cache_valid_time: 0`) as a pre_task here: https://github.com/aruhier/ansible-role-unbound/blob/master/molecule/default/converge.yml ?

Reply:
You might be right.
This is working on my laptop during my tests though. I'm trying your proposed solution with eade370.

For the APT Cache, I'll always advocate for up-to-date systems :grin: 
You can also try to pull a no-longer existing library if your cache is not updated, causing errors.
But both points are valid, let's see if it works with a `pre_tasks`.

BTW, the CI is quite slow, could be because `travis-ci.org` is deprecated in favor of `travis-ci.com`?

Reply:
Well, normally the outdated libraries are still hosted on the repositories, and at least it will just fail and it's the responsibility of the user to do an apt update and upgrade.

Thanks for your commit, I'm waiting for the CI to run to validate it. And about the fact that it's slow, I don't really know. I'm waiting for them to merge my account with travis-ci.com.

Reply:
Seems to have worked :+1: